roles/common/tasks/ssl.yml

```diff
@@ -76,13 +76,26 @@
     - /usr/local/lib/python2.7/dist-packages/requests/cacert.pem
   when: os == 'ubuntu' and ansible_distribution_version == "12.04"
 
-- name: force our ssl cert for python libs on precise
+- name: force our ssl cert for python libs on centos
   file: src=/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt dest={{ item }} owner=root
         mode=0644 state=link force=yes
   with_items:
     - /usr/lib/python2.7/site-packages/httplib2/cacerts.txt
     - /usr/lib/python2.7/site-packages/requests/cacert.pem
-  when: os == 'rhel'
+  when:
+    - os == 'rhel'
+    - openstack_install_method != "distro"
+
+- name: force our ssl cert for python libs on rhel
+  file: src=/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt dest={{ item }} owner=root
+        mode=0644 state=link force=yes
+  with_items:
+    - /opt/openstack/base/lib/python2.7/site-packages/requests/cacert.pem
```

Comment:
shouldn't we be using basevenv_lib_dir variable
also, seems like we have facts or variable defined for ca-bundle, etc.